### PR TITLE
Fix catch sizing

### LINKED ISSUE #2063: osu!catch needs scale adjustment to match osu-stable
screenshot_8

the catcher and fruits are huuge, not giving enough time to see where they're coming from

### PR DESCRIPTION
**Pull request by peppy**
- Fixes catcher size
- Reduces fruit sizes
- Makes it easier to scale 
- Adjusts the vertical position of the catcher (aka playfield scale) to match stable

Resolves #2063.

### CODE REVIEW

File: osu.Game.Rulesets.Catch/UI/CatcherArea.cs
Comment (smoogipoo) [reply osu.Game.Rulesets.Catch/UI/CatcherArea.cs]: I mean the old code had the same issue but you should look into it because this is probably not intended.

Comment (peppy) [reply osu.Game.Rulesets.Catch/UI/CatcherArea.cs]: Last I checked it was working as expected.

Comment (smoogipoo) [reply osu.Game.Rulesets.Catch/UI/CatcherArea.cs]: So I guess this is not being looked at further. I'm going to go ahead and merge this because the previous implementation had the same issue as far as I can see.

**Review by smoogipoo — DISMISSED** [review]

**Comment by peppy** [comment]
As this is June milestone I haven't yet checked into it. I will look at it before it is merged. Don't merge yet (and please check the may milestone PRs first).

**Comment by smoogipoo** [comment]
This is happening within the next 2 minutes.

**Comment by peppy** [comment]
Should be good to go

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**
